README.rst

```diff
@@ -148,11 +148,21 @@ Parsed ConfigDir
       "certificate": "<binary>",
       "host": "mail.mailgun.com",
       "password": "$3cr3t",
-      "port": "542",
+      "port": "542"
     }
   }
 
 
+Python Usage
+------------
+
+.. code-block:: python
+
+  from configdir import configdir
+  config = configdir()
```

Comment:
I decided against `import configdir; configdir()` because that was a lot of magic.